### PR TITLE
Fix charmap windows error and add test examples

### PR DESCRIPTION
**Pull request by cristim67**

### CODE REVIEW

File: test_all_examples.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+	run.test_example_repo("typescript", "crud-app", rootPath)
+	run.test_example_repo("typescript", "getting-started", rootPath)
+	run.test_example_repo("typescript", "hello-world", rootPath)
+	run.test_example_repo("typescript", "multiversx", rootPath)
+	run.test_example_repo("typescript", "quiz-app-prisma", rootPath)
+	run.test_example_repo("typescript", "shopping-cart", rootPath)
+	run.test_example_repo("typescript", "todo-list-angular", rootPath)
+	run.test_example_repo("typescript", "todo-list-flutter", rootPath)
+	run.test_example_repo("typescript", "todo-list", rootPath)
+	run.test_example_repo("typescript", "webhook", rootPath)
```
Comment (costinsin) [inline test_all_examples.py:16]: 1. variables in python are `snake_case`
2. it's better to define an array of examples and then loop through them, than copy paste the function multiple times.

```suggestion
    root_path = os.getcwd()
    examples = [
        { language: "typescript", app: "crud-app" },
        { language: "typescript", app: "getting-started" },
        { language: "typescript", app: "hello-world" },
        { language: "typescript", app: "multiversx" },
        { language: "typescript", app: "quiz-app-prisma" },
        { language: "typescript", app: "shopping-cart" },
        { language: "typescript", app: "todo-list-angular" },
        { language: "typescript", app: "todo-list-flutter" },
        { language: "typescript", app: "todo-list" },
        { language: "typescript", app: "webhook" },
    ]

    for example in examples:
        run.test_example_repo(example.language, example.app, root_path)
```

File: test_example_repo.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+	process = genezio_local()
+
+	assert process != None, "genezio local returned None"
+	kill_process(process)
+	print("Test passed!")
+
+
+# Test order matters because the commands are having side effects.
+if __name__ == '__main__':
+	test_example_repo("typescript", "webhook", os.getcwd())
```
Comment (costinsin) [inline test_example_repo.py:30]: Is this intended to only test `webhook`?

File: test_example_repo.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+import os
+from genezio import genezio_deploy, genezio_login, genezio_local
+from utils import kill_process
+
+
+def test_example_repo(language: str, repoNameExample: str, path: str):
+	print("Starting {} test...".format(language + " " + repoNameExample))
+	token = os.environ.get('GENEZIO_TOKEN')
+
+	os.chdir(os.path.join(path, "projects", "examples", "genezio-examples", language, repoNameExample))
```
Comment (costinsin) [inline test_example_repo.py:12]:
```suggestion
def test_example_repo(language: str, repo_name_example: str, path: str):
	print("Starting {} test...".format(language + " " + repo_name_example))
	token = os.environ.get('GENEZIO_TOKEN')

	os.chdir(os.path.join(path, "projects", "examples", "genezio-examples", language, repo_name_example))
```

**Review by costinsin — CHANGES_REQUESTED** [review]

File: .gitmodules [hunk 4]
```diff
@@ -0,0 +1,3 @@
+[submodule "projects/examples/genezio-examples"]
+	path = projects/examples/genezio-examples
```
Comment (radu1122) [inline .gitmodules:2]: here should be added an automatic flow to get all the time the latest updates from the genezio-examples

**Review by costinsin — APPROVED** [review]

**Pull request merged**